transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebAcceptProcessor.java

```diff
@@ -100,7 +100,7 @@ protected void flushInternal(final WsebSession session) {
         }
 
         // get write request queue and process it
-        final WriteRequestQueue writeRequestQueue = session.getWriteRequestQueue();
+        final WriteRequestQueue writeRequestQueue = session.getTransportSession().getWriteRequestQueue();
```

Comment:
I think the same as it involves extensive changes. Let us get the current changeset in and later worry about other things.